websites/views_test.py

```diff
@@ -577,7 +577,27 @@ def test_websites_autogenerate_name(drf_client):
     )
     website_title = "My Title"
     website_short_id = "my-title"
-    slugified_title = slugify(website_title)
+    slugified_title = slugify(website_title, allow_unicode=True)
+    resp = drf_client.post(
+        reverse("websites_api-list"),
+        {"title": website_title, "short_id": website_short_id, "starter": starter.id},
+    )
+    assert resp.status_code == status.HTTP_201_CREATED
+    assert resp.data["name"] == slugified_title
+    assert resp.data["short_id"] == website_short_id
+
+
+def test_websites_autogenerate_name_unicode(drf_client):
+    """The autogenerated name should be compatible with unicode characters"""
+    superuser = UserFactory.create(is_superuser=True)
+    drf_client.force_login(superuser)
+    starter = WebsiteStarterFactory.create(
+        source=constants.STARTER_SOURCE_GITHUB,
+        status=constants.WebsiteStarterStatus.ACTIVE,
+    )
+    website_title = "テスト ページ"
+    website_short_id = "テスト_ページ"
```

Comment:
It seems like GitHub doesn't allow unicode and our Studio UI forbits it as well for `short-id`. We should probably not test this here, and maybe add a test to ensure short-id is not unicode compatible.

<img width="779" alt="Screenshot 2024-02-07 at 5 31 12 PM" src="https://github.com/mitodl/ocw-studio/assets/71316217/c0e28eb9-00b3-4bff-9f0f-8a93bcf60957">